content/blog/2021-12-10-recap-code-for-germany-community-summit-2021

```diff
@@ -0,0 +1,50 @@
+---
+title: "Das war der Code for Germany Summit 2021"
+author: "Sonja Fischbauer"
+date: '2021-12-31T17:20:00+01:00'
+excerpt: Knapp 20 Ehrenamtliche und hauptamtliche Menschen aus dem Code for Germany Netzwerk trafen sich am 27. November 2021 in Berlin, zum ersten Community Summit seit Pandemiebeginn. 
+topic: event
+images:
+- - imgname: cfgsummit21-gruppenfoto.jpg
+  attribution: Foto von Wolfram Eberius, Lizenz [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/)
+og_image: cfgsummit21-gruppenfoto.jpg
+type: blog
+---
+
+# Das war der Code for Germany Summit 2021
+
+Knapp 20 Ehrenamtliche und hauptamtliche Menschen aus dem Code for Germany Netzwerk trafen sich am 27. November 2021 in Berlin, zum ersten Community Summit seit Pandemiebeginn. 
+
+
+Neben einem kleinen Meetup im Rahmen des [Hackday Moers](https://www.codeforniederrhein.de/hackday-2021/) war der Summit das erste bundesweite vor-Ort-Treffen der Code for Germany Community seit der [Erstauflage im Jahr 2019](https://codefor.de/blog/cfg-community-summit-2019/). 
+
+Dieses Jahr kamen insgesamt 16 Teilnehmende, davon 13 Ehrenamtliche am Samstag, 27. November in den Räumen des bUm am Paul-Lincke-Ufer in Berlin unter 2G+ Regelung zusammen. 
+
+Am Abend davor hatte ein Großteil der Leute schon die Möglichkeit genutzt und sich bei Pizza, Limo und anderen Drinks getroffen. Am Tag des Summits starteten wir Vormittags mit einer ausgiebigen Vorstellungsrunde und Berichten aus den lokalen Labs: Ehrenamtliche Communitymitglieder waren aus Hamburg, Berlin, Köln, Moers/ Niederrhein, Leipzig, Chemnitz, Kaiserslautern, Nürnberg und Ulm angereist! Die angeregten Gespräche über die innere Struktur und die äußere Wirkung von Code for Germany begleiteten uns bis über die Mittagspause.
+
+  - imgname: cfgsummit21-austausch.jpg 
```

Comment:
What happened here? I think this line needs to be removed

Reply:
Or do you want to display the picture here? In this case , I think the syntax is `![description](link)`